functions/views/pages/fuse19.ejs

```diff
@@ -15,50 +15,66 @@
-		<div class="overlay"></div>
-		<div class="gtco-container">
-			<div class="row">
-				<div class="col-md-8 col-md-offset-2 text-center">
-					<div class="display-t">
-						<div class="display-tc">
-							<h1 class="animate-box" data-animate-effect="fadeInUp">Fuse 2019</h1>
-							<h2 class="animate-box" data-animate-effect="fadeInUp">Connect Your Ideas</h2>
+			<header id="gtco-header about" class="gtco-cover gtco-cover-xs" role="banner" style="background-image:url(images/img_bg_1.jpg);">
+				<style>
+		
+					.round {
+					  display: round;
+					  width: 30%;
+					  border: none;
+					  background-color: rgba(38, 0, 255, 0.938);
+					  color: white;
+					  padding: 14px 28px;
+					  font-size: 16px;
+					  cursor: pointer;
+					  text-align: center;
+					}
+					
+					.round:hover {
+					  background-color: #ddd;
+					  color: green;
+					}
+					</style>			
+				<div class="overlay"></div>
+				<div class="gtco-container">
+					<div class="row">
+						<div class="col-md-8 col-md-offset-2 text-center">
+							<div class="display-t">
+								<div class="display-tc">
+									<h1 class="animate-box" data-animate-effect="fadeInUp">Fuse 2019</h1>
+								</div>
+							</div>
 						</div>
 					</div>
 				</div>
-			</div>
-		</div>
-	</header>
+			</header>
 
 	<div id="gtco-history" class="gtco-section border-bottom animate-box">
 		<div class="gtco-container">
 				
 			<div class="row">
 				<div class="col-md-8 col-md-offset-2 text-center gtco-heading">
 					<h2>Fuse 2019</h2>
-					<p>YouthComputing is hosting Fuse 2019 at Westwood Community High School on November 25. Fuse is an open house showcasing various technologies and promotes science, technology, engineering, and mathematics.</p>
-				<h3>Pictures from the event: </h3>
-				<a href="https://woodbuffalo.snapd.com/events/view/1104996">https://woodbuffalo.snapd.com/events/view/1104996</a>
-				<a href="https://www.dropbox.com/sh/94n5qcy9r6pcxum/AAAH7fc36wikvLBEzGBa4FdAa?dl=0">https://www.dropbox.com/sh/94n5qcy9r6pcxum/AAAH7fc36wikvLBEzGBa4FdAa?dl=0</a>
+					<p>YouthComputing is hosting Fuse 2019 at Westwood Community High School on December 21. Fuse is an oppurtunity for youth to showcase various technological inventions and it also promotes science, technology, engineering, art and mathematics.</p>
 				</div>
-			</div>
-			
-			
-			
-				<div class="row">
+
+
 				<div class="col-md-12">
-					<h3>Information about the Different Booths</h3>
+				<div class="col-md-8 col-md-offset-2 text-center gtco-heading">
+					<h3>Below are some of the pictures from the event: </h3>
+					<div class="item">
+						<img src="images/fuse4.jpg" width="100%" alt="Booths">
+					</div>
+					<p></p>
+					<p></p>
+					<p> <a href="https://drive.google.com/drive/folders/13lqEoBKIoQVqhJMRkmJdQJf0vyRAT4l3?usp=sharing" class="round">More Pictures From Fuse 2019</a></p>
```

Comment:
this link is to old drive, update the link

Reply:
Admin needs to enable link sharing out of the YC organization

Reply:
Talked with Krish and we cannot share outside of our organization, so I just kept the old link